flocker/node/agents/cinder.py

```diff
@@ -283,14 +299,9 @@ def create_volume(self, dataset_id, size):
         }
         action_type = u"blockdevice:cinder:create_volume"
         with start_action(action_type=action_type):
-            # There could be difference between user-requested and
-            # Cinder-created volume sizes due to several reasons:
-            # 1) Round off from converting user-supplied 'size' to 'GiB' int.
-            # 2) Cinder-specific size constraints.
-            # XXX: Address size mistach (see
-            # (https://clusterhq.atlassian.net/browse/FLOC-1874).
+            allocated_bytes = allocated_size(self.allocation_unit(), size)
```

Comment:
The deployer is doing this, so I don't think IBlockDeviceAPIs should be required to (or need to!).